packages/tabs/tabs.scss

```diff
@@ -0,0 +1,52 @@
+@use "~@fremtind/jkl-core/jkl";
+
+$tab-text-color: jkl.$color-granitt;
+$tab-text-color--inverted: jkl.$color-snohvit;
+
+$tab-indicator-color: jkl.$color-granitt;
+$tab-indicator-color--inverted: jkl.$color-snohvit;
+
+$tab-hover-color: jkl.$color-stein;
+$tab-hover-color--inverted: jkl.$color-svaberg;
+
+@include jkl.helper-light-mode-variables {
+    --tab-text-color: #{$tab-text-color};
+    --tab-indicator-color: #{$tab-indicator-color};
+    --tab-hover-color: #{$tab-hover-color};
+}
+
+@include jkl.helper-dark-mode-variables {
+    --tab-text-color: #{$tab-text-color--inverted};
+    --tab-indicator-color: #{$tab-indicator-color--inverted};
+    --tab-hover-color: #{$tab-hover-color--inverted};
+}
+
+.jkl-tablist {
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    padding: 0;
+    margin: 0;
+    border-bottom: 1px jkl.$color-svaberg solid;
+    width: 100%;
+}
+
+.jkl-tab {
+    @include jkl.text-style("body");
+    border: none;
+    background-color: transparent;
+    padding: jkl.$spacing-m;
```

Comment:
Denne paddingen "stacker" om vi har tabs inni tabs. Vet ikke om det er ønsket effekt?

![image](https://user-images.githubusercontent.com/1223410/140283469-f775ae34-a8c5-428f-bf3e-ffb587f31072.png)

Kan eventuelt offsettes med negativ margin for tabs-i-tabs.

```scss
& .jkl-tab {
    margin: -#{jkl.$spacing-m};
    padding: jkl.$spacing-m;
}
```

Reply:
Det er faktisk TabPanel i eksempelet som har en jkl-spacing-m--all på seg.

Reply:
Jeg vil si at det er riktig at de får paddingen rundt seg. Hvis ikke vil det jo se ut som det er to forskjellige (altså helt separate) tab-grupper.

Det at dette er et issue i det hele tatt viser jo kanskje hvor forsiktig man skal være med flere nivåer av tabs generelt.